ambry-account/src/main/java/com/github/ambry/account/MySqlAccountService.java

```diff
@@ -249,14 +249,16 @@ public void updateAccounts(Collection<Account> accounts) throws AccountServiceEx
     infoMapLock.readLock().lock();
     try {
       AccountInfoMap accountInfoMap = accountInfoMapRef.get();
-      if (accountInfoMap.hasConflictingAccount(accounts)) {
+      //TODO: Temporarily disabling version check with CompositeAccountService enabled. Remove it after moving to MySql only.
+      if (accountInfoMap.hasConflictingAccount(accounts, true)) {
```

Comment:
Can we make ignoreAccountVersion a mysql config property?  That way we won't need to change the code later.  Default can be false and override in config when composite AS is used.

Reply:
Right, config seems to be better. This is one-off migration and we can keep the default value false after it's complete.